ios/RN/BarcodeDetectorManagerMlkit.m

```diff
@@ -1,5 +1,6 @@
 #import "BarcodeDetectorManagerMlkit.h"
 #import <React/RCTConvert.h>
+#import <React/RCTLog.h>
```

Comment:
question: why was this lib imported?

Reply:
that was for logging. I should have removed it before pushing, my mistake. I'll fix it shortly.